internal/pkg/config/v1/config.go

```diff
@@ -154,6 +154,19 @@ func (c *Config) DecryptContextStates() error {
 	return c.Validate()
 }
 
+func (c *Config) DecryptCredentials() error {
```

Comment:
This seems a bit wasteful. I don't think we'd ever use all of the stored API keys in a single command, right?

Reply:
I think it's ok for apikeys for contexts. the thing is we might need to encrypt/decrypt context apikeys even when they're not the active/current one. But you have a good point. For stored kafka-cluster api-keys, I'll only decrypt them when needed.

Reply:
> we might need to encrypt/decrypt context apikeys even when they're not the active/current one

Really? What's an example of that?

Reply:
e.g. creating a new api key for kafka cluster